tracer.go

```diff
@@ -210,3 +210,16 @@ func (t *tracerImpl) Join(operationName string, format interface{}, carrier inte
 	}
 	return nil, opentracing.ErrUnsupportedFormat
 }
+
+// GetGlobalTracerOptions attempts to get the basictracer.Options
+// struct provided to the global Tracer.  Returns the options and true
+// if the global Tracer is a basictracer implementation; returns
+// false otherwise.
+func GetGlobalTracerOptions() (Options, bool) {
```

Comment:
but why do you need access to Recorder? Regular application code should not need it. If you need it in tests, then presumably those same tests are actually in charge of creating the instance of the Recorded in the first place, so they could just keep that ref.